scripts/gen_vmlinux_h.sh

```diff
@@ -85,8 +85,26 @@ generate_vmlinux_for_arch() {
     echo "" > ${LOG}
     echo "Writing compile logs to ${LOG}"
 
+    rm -f .config.orig vmlinux
+    if [ -e .config ]; then
+        cp .config .config.orig
+    else
+        make ARCH=${ARCH} CROSS_COMPILE=${CROSS_COMPILE} KCFLAGS=-Wno-error defconfig &>> ${LOG}
+        echo CONFIG_DEBUG_INFO_REDUCED=n >>.config
+        echo CONFIG_DEBUG_INFO_DWARF4=y >>.config
+        echo CONFIG_BPF_SYSCALL=y >>.config
+        echo CONFIG_DEBUG_INFO_BTF=y >>.config
+        echo CONFIG_BPF_JIT=y >>.config
+        echo CONFIG_SCHED_CLASS_EXT=y >>.config
+        echo CONFIG_CGROUP_SCHED=y >>.config
```

Comment:
I think we are missing `CONFIG_BPF_EVENTS=y` here. Without it I get the following error building on [arm64](https://build.opensuse.org/package/live_build_log/home:flonnegren:branches:Base:System/scx/openSUSE_Factory/aarch64):

```
...
[  316s]       3: src/bpf/main.bpf.c:1146:64: error: incomplete definition of type 'struct bpf_perf_event_data'
[  316s]           1146 |         event->event.perf_sample.instruction_pointer = PT_REGS_IP(&ctx->regs);
...
```